.prettierrc.json

```diff
@@ -0,0 +1,9 @@
+{
+	"tabWidth": 2,
+	"useTabs": true,
+	"printWidth": 80,
+	"semi": true,
+	"trailingComma": "es5",
+	"singleQuote": true,
```

Comment:
@sathwik-14: I have given sample, we can move this singleQuote to false as well.

Reply:
should we follow same options to our generated code? can check format.js under utils/

Reply:
Makes sense, made the necessary changes